Update Teleport Cloud FAQ (#12496)
gravitational/teleport: docs/pages/cloud/faq.mdx

It's a Teleport-managed S3 bucket with AWS-managed keys.
Currently there is no way to provide your own bucket.

## How long will Teleport Cloud retain my data?

> See our documentation on [data retention](./architecture.mdx#data-retention)

Reviewer: ```suggestion
See our documentation on [data retention](./architecture.mdx#data-retention).
```

## How can I access the tctl admin tool?

> Find the appropriate download at [Teleport Cloud Downloads](./downloads.mdx). Use the enterprise version of `tctl` and `tsh`.

Reviewer: ```suggestion
Find the appropriate download at [Teleport Cloud Downloads](./downloads.mdx). Use the Enterprise version of `tctl`.
```

My understanding is that there is no separate Enterprise version of `tsh` ([source](https://github.com/gravitational/teleport/pull/10920#discussion_r848898440)).

> After downloading the tools, first log into your cluster using `tsh`, then use `tctl` remotely:

Reviewer: ```suggestion
After downloading the tools, first log in to your cluster using `tsh`, then use `tctl` remotely:
```

$ tctl nodes add --ttl=5m --roles=node,proxy --token=$(uuid)
```

## Is an independent security audit available?

> A security audit has been completed and is available [here](https://goteleport.com/resources/audits/teleport-cloud-security-audit-2021/).

Reviewer: ```suggestion
A security audit has been completed and is available at our [Audit Report 2021](https://goteleport.com/resources/audits/teleport-cloud-security-audit-2021/) page.
```
Added a suggestion to make this a bit clearer (https://www.smashingmagazine.com/2012/06/links-should-never-say-click-here/#here-conceals-what-users-are-clicking).

## Where does Teleport Cloud run?

> Teleport Cloud is deployed within AWS, using S3 and DynamoDB for storage with AWS managed encryption keys. All data is currently hosted in us-west-2.

Reviewer: ```suggestion
Teleport Cloud is deployed within AWS, using S3 and DynamoDB for storage with AWS-managed encryption keys. All data is currently hosted in us-west-2.
```

The ability to download recordings for offline viewing will be available in a future release.

> ## Is there a way to forward Teleport Cloud audit events to my companies internal SEIM?

Reviewer 2: I think this should be `company's`.  And probably `SIEM` should be spelled out too along with the acronym.

Author: Changed both of these in f0a79a209cb3678b0de09d35986ae2ce740001d.  Thanks!

The ability to download recordings for offline viewing will be available in a future release.

> ## Is there a way to forward Teleport Cloud audit events to my company's internal Security Event and Incident Managment (SEIM)?

Reviewer 3: I traditionally see "SIEM" used more than "SEIM", but maybe I'm behind the times?

Author: Good catch. Changed in [0e596c3](https://github.com/gravitational/teleport/pull/12496/commits/0e596c3bad29e200e8a27be85236450f7babebec).
